packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -166,6 +166,24 @@ export default /* @ngInject */ ($stateProvider) => {
           servers: [server],
         });
       },
+
+      vpsStateInfo: /* @ngInject */ (stateVps) => {
+        const state = stateVps.state.toLowerCase();
+        return {
+          state,
+          stateGroup: {
+            error: ['maintenance', 'stopped', 'stopping'].includes(state),
+            warning: [
+              'backuping',
+              'installing',
+              'rebooting',
+              'upgrading',
+            ].includes(state),
+            success: ['running'].includes(state),
```

Comment:
I'm not sure this should be store in the routing, maybe a constant file and some testing or a utility function

Reply:
Fixed.